### DISCUSSION TITLE
Do you use Windows for hacking on Copier?

### DISCUSSION THREAD

**Discussion by yajo** [copier-org/copier#757] [Polls]
In https://github.com/copier-org/copier/pull/696 I'm attempting to leverage [precommix](https://gitlab.com/moduon/precommix) for all the linting and formatting job.

Precommix is a new tool I developed that puts an end to the problem of pre-commit hook installation. I just got fed up with hooks that suddenly started failing to install in random machines.

It's not (yet) a big problem here in Copier, but I think it's good that Copier gives an example in this regard because Copier is used in some organizations to keep those configurations sane and updated.

However, right now precommix only supports x86_64-linux. And it could support other arches of Linux and Mac, but it will never work on Windows (without WSL).

I wouldn't like to fix a problem by introducing a worse one, so here's the poll.

**Comment by pawamoy** [comment]
You could probably ping previous contributors to get more votes 🙂

**Comment by rafalkrupinski** [comment]
- [x] macos
- [x] Linux x86
